commands/login-ci.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var Command = require('../lib/command');
+var chalk = require('chalk');
+var utils = require('../lib/utils');
+var login = require('../lib/login');
+
+module.exports = new Command('login:ci')
+  .description('sign into Firebase ')
+  .action(function(options) {
+    if (utils.getInheritedOption(options, 'nonInteractive')) {
+      return utils.reject('Cannot run login in non-interactive mode. Pass ' + chalk.bold('--token') + ' instead.', {exit: 1});
```

Comment:
Again I think this needs a reword, but I'm struggling to find something that sounds right. Any ideas?